extensions/vertx-http/deployment/src/main/java/io/quarkus/vertx/http/deployment/HttpSecurityProcessor.java

```diff
@@ -107,6 +109,12 @@ SyntheticBeanBuildItem initBasicAuth(
             configurator.defaultBean();
         }
 
+        if (capabilities.isPresent(Capability.JWT)) {
```

Comment:
I'm not sure about this code - and specifically about JWT - did you mean `Form` ? Capabilities should likely not be used as well, it is either `buildTimeConfig.auth.form.enabled` or `buildTimeConfig.auth.basic.enabled` (there is also a block there which defaults to `basic`)

Reply:
JWT is also basic, so if I do this in the JWT extension, this one also runs, adding normal basic. Not sure how else to do this. Either in the extensions, but then I need to check the config key to see if form is enabled, or here then I need to check the capabilities for JWT.  Which one do you prefer ? Or do you have another option ?

Reply:
Hold on let me double check if I can do with in the JWT extension....

Reply:
OK, ignore what I said, it's now done without checking capabilities in `HttpSecurityProcessor`